scripts/pr-infra-compose.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+# This script enables the deployment of a dedicated app instance using the
+# following inputs:
+#   * PR Number: Specify the pull request number for the instance you wish to
+#       run.
+#   * Infrastructure Backup: Provide a backup of the infrastructure to load
+#       into the app.
+#
+# Key Benefits:
+#   * Isolated Environment: The new instance operates independently of the
+#       development environment, allowing both to run in parallel without
+#       interference.
+#   * Faster Reviews: The script pulls Docker images generated by CI,
+#       eliminating the need to rebuild everything from scratch, which
+#       accelerates the review process.
+
+
+set -e
+
+# Usage
+
+if  {
+        [ "$#" -eq 3 ] &&
+        [ "$2" != "up-and-load-backup" ]
+    } ||
+    {
+        [ "$#" -eq 2 ] &&
+        {   [ "$2" != "up" ] &&
+            [ "$2" != "down" ] &&
+            [ "$2" != "down-and-clean" ]
+        }
+    }; then
+
+    echo "Usage: $0 pr-number (up | up-and-load-backup [osrd.backup] | down | down-and-clean )" >&2
+    exit 1
+
+fi
+
+# Change ports in needed files
+export PR_NB="pr-$1"
+
+if [ "$2" = "up" ] || [ "$2" = "up-and-load-backup" ]; then
+
+    # Load the docker instance
+    docker compose \
+        -p "osrd-pr-tests" \
+        -f "docker/docker-compose.pr-tests.yml" \
+        up -d
```

Comment:
You should only do this up when doing load backup, and should should only up postgres

Reply:
I had to add redis for the loading script to work